### PR TITLE
Mainnet configuration

### PR DESCRIPTION
**Pull request by talekhinezh**
* Configure minimum stake of 100
* Configure 0 emissions for epoch 1
* Configure 2 week period for epoch 1
* Implement 98% completed proposals requirement for rewards

### CODE REVIEW

File: radixdlt-engine/src/main/java/com/radixdlt/atommodel/system/scrypt/EpochUpdateConstraintScrypt.java [hunk 1]
```diff
@@ -139,41 +146,56 @@ public ReducerState process(ShutdownAll<ValidatorBFTData> i) throws ProcedureExc
...
 				throw new IllegalStateException();
 			}
-			var numProposals = proposalsCompleted.remove(k);
-			var nodeEmission = rewardsPerProposal.multiply(UInt256.from(numProposals));
+			var bftData = validatorBFTData.remove(k);
+			if (bftData.proposalsCompleted() + bftData.proposalsMissed() == 0) {
+				return next();
+			}
+
+			var percentageCompleted = bftData.proposalsCompleted() * 10000
```
Comment (michalzajda) [inline radixdlt-engine/src/main/java/com/radixdlt/atommodel/system/scrypt/EpochUpdateConstraintScrypt.java:172]: ok, so here we use data from ValidatorBFTData

File: radixdlt-engine/src/main/java/com/radixdlt/atommodel/system/construction/NextEpochConstructorV3.java [hunk 2]
```diff
@@ -137,37 +143,52 @@ public void construct(NextEpoch action, TxBuilder txBuilder) throws TxBuilderExc
...
 		var preparingStake = new TreeMap<ECPublicKey, TreeMap<REAddr, UInt256>>(KeyComparator.instance());
-		for (var e : proposals.entrySet()) {
+		for (var e : validatorBFTData.entrySet()) {
 			var k = e.getKey();
-			var numProposals = e.getValue();
+			var bftData = e.getValue();
+			if (bftData.proposalsCompleted() + bftData.proposalsMissed() == 0) {
+				continue;
+			}
+			var percentageCompleted = bftData.proposalsCompleted() * 10000
```
Comment (michalzajda) [inline radixdlt-engine/src/main/java/com/radixdlt/atommodel/system/construction/NextEpochConstructorV3.java:161]: 👍

**Review by michalzajda — APPROVED** [review]

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/DispatcherModule.java [hunk 3]
```diff
@@ -195,7 +196,7 @@ public void configure() {
 		bind(new TypeLiteral<EventDispatcher<EpochLocalTimeoutOccurrence>>() { }).toProvider(
 			Dispatchers.dispatcherProvider(
 				EpochLocalTimeoutOccurrence.class,
-				(t, a) -> String.format("Timeout{epoch=%s round=%s leader=%s nextLeader=%s count=%s}",
+				(t, a) -> String.format("bft_timout{epoch=%s round=%s leader=%s nextLeader=%s count=%s}",
```
Comment (siy) [inline radixdlt-core/radixdlt/src/main/java/com/radixdlt/DispatcherModule.java:199]: It's a typo or all names are intentionally formatted as "xxx_xxxxxx" (i.e. 3 symbols followed by 6 symbols and separated by underscore)?

**Review by siy — APPROVED** [review]
LGTM with one minor comment

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/DispatcherModule.java (hunk 3 above)
Comment (talekhinezh) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/DispatcherModule.java:199]: Yup, exactly trying to make the log stream more readable/parsable

**Pull request merged**